styles/style.css

```diff
@@ -1,8 +1,33 @@
+* {
+  font-family: sans-serif;
+}
+
+img {
+  max-width: 100%;
+}
+
 /* put something in to hide the nav bar for smaller screens */
-/* something like this:
-@media (min-width: 800px) {
+/* something like this: */
+@media (max-width: 800px) {
```

Comment:
this website should be mobile-first, which means that you want the styles for your mobile website to be outside the media queries and the styles for larger websites would be inside the media query.